pytorch_lightning/callbacks/model_checkpoint.py

```diff
@@ -45,7 +45,15 @@
 warning_cache = WarningCache()
 
 
-class ModelCheckpoint(Callback):
+class Checkpoint(Callback):
```

Comment:
why not split this out into its own file?

Reply:
It is now